### PR TITLE
Refactor DistSQL RAL assertion logic of integration test

### PR DESCRIPTION
**Pull request by zhaojinchao95**
For #20867

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ral/BaseRALIT.java [hunk 1]
```diff
@@ -106,23 +115,27 @@ private void assertMetaData(final ResultSetMetaData actual, final Collection<Dat
...
     }
     
-    private void assertRow(final ResultSet actual, final ResultSetMetaData actualMetaData, final DataSetRow expected) throws SQLException {
+    private void assertRow(final ResultSet actual, final Collection<String> notAssertionColumns, final ResultSetMetaData actualMetaData, final DataSetRow expected) throws SQLException {
         int columnIndex = 1;
         for (String each : expected.splitValues("|")) {
             String columnLabel = actualMetaData.getColumnLabel(columnIndex);
-            assertObjectValue(actual, columnIndex, columnLabel, each);
-            columnIndex++;
+            if (notAssertionColumns.contains(columnLabel)) {
```
Comment (LeeGuoPing) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ral/BaseRALIT.java:133]: if (!notAssertionColumns.contains(columnLabel)) {
            assertObjectValue(actual, columnIndex, columnLabel, each);
        }
        columnIndex++;

Comment (zhaojinchao95) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ral/BaseRALIT.java]: Thx, i think your suggest are right.

Comment (LeeGuoPing) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ral/BaseRALIT.java]: `Suggest `is a verb，use `suggestion` here may better. Do you think so?

**Comment by zhaojinchao95** [comment]
image

**Review by LeeGuoPing — APPROVED** [review]
Good job

**Pull request merged**